### ISSUE TITLE
Saving loses escaped characters

### ISSUE THREAD

**Issue by madduck** [james-fray/tab-reloader#118]
If I use a `url` field in a reload policy, such as:

```
  "example.com": {
    "url": "\/foo"
  }
```

then saving these rules will turn `\/` into `/`, which means the regular expression actually won't match anymore.

Note that this is an issue of it's own, though my suggestions in #65 and especially #117 will also take care of this, and it might just be better to use the time there…

**Comment by james-fray** [comment]
how about `\\/`?

**Comment by james-fray** [comment]
Note that currently, the matching uses URLPattern matching, which is a bug. I just pushed a fix

**Comment by madduck** [comment]
> how about `\\/`?

Yes, that works, but it's even more confusing, and also unnecessary. When specifying regexps for URLs, `/` should not be a character you'd need to escape, let alone double-escape.
